packages/@glimmer/integration-tests/test/updating-test.ts

```diff
@@ -1893,8 +1893,11 @@ class UpdatingTest extends RenderTest {
     this.rerender({ list: ['initial', 'update'] });
     this.assertHTML(`<div>initial</div><div>update</div>`);
 
+    this.rerender({ list: ['initial'] });
+    assert.equal(destroyCount, 1, 'destroy was called for removed item');
+
     this.rerender({ list: [] });
-    assert.equal(destroyCount, 2, 'both list items were correctly destroyed');
+    assert.equal(destroyCount, 1, 'remaining list item was correctly destroyed');
```

Comment:
I believe this should still be 2:

```suggestion
    assert.equal(destroyCount, 2, 'remaining list item was correctly destroyed');
```

Reply:
You're totally right. Fixed and rebased.